gnomad_qc/v4/annotations/generate_variant_qc_annotations.py

```diff
@@ -317,16 +326,49 @@ def run_generate_sib_stats(
 
 
 def get_variant_qc_annotation_resources(
-    test: bool, overwrite: bool
+    test: bool,
+    overwrite: bool,
+    large_n_alleles: Optional[bool] = None,
+    combine_compute_info: bool = False,
 ) -> PipelineResourceCollection:
     """
     Get PipelineResourceCollection for all resources needed in the variant QC annotation pipeline.
 
     :param test: Whether to gather all resources for testing.
     :param overwrite: Whether to overwrite resources if they exist.
+    :param large_n_alleles: Whether to use a temporary info TableResource for results
+        of only sites with a large number of alleles. If False, a temporary info
+        TableResource for results of only sites with a smaller number of alleles is
+        used. By default, when None, the finalized info ht is used instead of a
+        temporary location.
```

Comment:
```suggestion
    :param large_n_alleles: Whether to use a temporary info TableResource for results. When 
    True, use temporary info TableResource for only sites with a large number of alleles. When
    False, use temporary info TableResource for only sites with a small number of alleles. When 
    None, the finalized info ht is used instead of a temporary location. Default is None.
```
I had to read through this a couple times to understand it. Since there are three options, I figured just spell them all out but no strong feelings.